test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -712,33 +712,31 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
     }
 
     It "Validate Invoke-WebRequest returns valid RelationLink property with absolute uris if Link Header is present" {
-        $command = "Invoke-WebRequest -Uri 'http://localhost:8080/PowerShell?test=linkheader&maxlinks=5'"
+        $uri = Get-WebListenerUrl -Test 'Link' -Query @{maxlinks = 5; linknumber = 2}
+        $command = "Invoke-WebRequest -Uri '$uri'"
         $result = ExecuteWebCommand -command $command
-        $result.Output.RelationLink.Count | Should BeExactly 2
-        $result.Output.RelationLink["next"] | Should BeExactly "http://localhost:8080/PowerShell?test=linkheader&maxlinks=5&linknumber=2"
-        $result.Output.RelationLink["last"] | Should BeExactly "http://localhost:8080/PowerShell?test=linkheader&maxlinks=5&linknumber=5"
+
+        $result.Output.RelationLink.Count | Should BeExactly 5
```

Comment:
it was like that before I didn't change them. Does it really matter though?

Reply:
I'd prefer `Should Be` because `BeExactly` is used for strings and my first thought was that here `Count` is a method in `RelationLink` which return a string.

Reply:
@iSazonov This appears to happen in more place. Lets leave it as is here and I will address them all as part of #5669 which I have updated.